app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -1,13 +1,11 @@
 package com.droidhats.campuscompass.views
 
-import android.Manifest
```

Comment:
According to Jenkins, the build fails after this commit. Is there maybe an import statement that you removed but is needed? 

@NicholasNagy could you check the error logs on Jenkins

Reply:
I had played with the settings to run the Jenkins agent automagically on start up, and it wasn't finding the API key. 
I set the home manually through the Jenkins config so it should work now